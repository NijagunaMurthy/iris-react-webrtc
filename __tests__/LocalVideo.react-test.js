'use strict';

import React from 'react';
import { LocalVideo } from '../src/index.jsx';
import renderer from 'react-test-renderer';

describe('LocalVideo', () => {
  it('renders correctly', () => {
    const testLocalConnection = {
      video: {
        index: 0,
        src: "https://xrtc.iris.me/video0",
      },
      audio: {
        index: 1,
        src: "https://xrtc.iris.me/audio1",
      }
    }
    const tree = renderer.create(
      <LocalVideo key="1" audio={testLocalConnection.audio} video={testLocalConnection.video} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
