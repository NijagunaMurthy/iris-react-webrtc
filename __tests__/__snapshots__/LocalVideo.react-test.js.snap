exports[`LocalVideo renders correctly 1`] = `
<div>
  <video
    autoPlay="1"
    id="localVideo0"
    src="https://xrtc.iris.me/video0" />
  <audio
    autoPlay="1"
    id="localAudio1"
    muted="true"
    src="https://xrtc.iris.me/audio1" />
</div>
`;
